app/components/partner/credit-transfer/index.scss

```diff
@@ -0,0 +1,121 @@
+.credit-transfer {
+  margin: 0 0.35rem;
+}
+
+.credit-transfer-btn {
+  border-radius: 50%;
+}
+
+.client {
+  display: grid;
+  grid-template-columns: auto auto;
+  align-items: center;
+  column-gap: 1rem;
+  justify-content: center;
+}
+
+.invalid-credits {
+  color: $red;
+  font-size: 0.8rem;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  justify-self: center;
+}
+
+.title {
+  text-transform: capitalize;
+  font-weight: bold;
+  color: rgba(#000, 0.7);
+  max-height: 92%;
+  font-size: 1.1rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  &.empty-title {
+    color: #424651;
+    font-size: 1em;
+    font-style: italic;
+  }
+}
+
+.credit-tranfer {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  row-gap: 1.5rem;
+}
+
+.transferable-credits {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #0063b2ff;
```

Comment:
Specifying colors randomly in component level has become unmaintainable. 

Let's restrain from not directly specifying any colors in components via css module classes, instead use color util classes.

Reply:
Agree, we may have to define color codes as variables, may be start adding while upgrading the ux.